src/validate/validate.ts

```diff
@@ -46,6 +47,7 @@ const VALIDATORS = {
     'light': validateLight,
     'sky': validateSky,
     'terrain': validateTerrain,
+    'projectionConfig': validateProjectionConfig,
```

Comment:
Should this be a different validation than "regular" object?

Reply:
Yes, 

validateProjectionConfig check a value of the type:

{ type: projection }

validateProjection check a value of the type:

projection

Reply:
Or is there's a regular regular object fallback I can use, that just check the `type`, so we don't need this?

Reply:
I don't know, worth a check, if not, then this is OK as is.